modules/core/src/main/scala/parser.scala

```diff
@@ -137,7 +137,7 @@ object Parser {
     (Name <~ keyword(":")) ~ Value
 
   lazy val VariableDefinitions =
-    parens(many(VariableDefinition)).token
+    parens(many(VariableDefinition <~ opt(keyword(",")))).token
```

Comment:
My understanding is that commas are whitespace in graphql so I think we need to solve this more generally. But this is ok for now.

Reply:
That's right. I need to add support for comments to enable schema (de) serialisation and I'll pick this up in the process.